main_spec.tex

```diff
@@ -110,7 +110,7 @@ \subsubsection{\textbf{SHPALLOC}}\label{subsec:shpalloc}
 \subsubsection{\textbf{SHPCLMOVE}}\label{subsec:shpclmove}
 \input{content/shpclmove.tex}
 
-\subsubsection{\textbf{SHPDEALLOC}}\label{subsec:shpdealloc}
+\subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdealloc}
```

Comment:
Oh, wow, I didn't think it was possible for me to like the Fortran bindings less..

Reply:
When I first saw `SHPDEALLC`, I actually corrected the spelling on the API, only to realize that it was correct and the TOC entry was wrong. :sob: